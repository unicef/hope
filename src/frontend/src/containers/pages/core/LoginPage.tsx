import { Button, Typography } from '@mui/material';
import * as React from 'react';
import { useTranslation } from 'react-i18next';
import { useLocation } from 'react-router-dom';
import styled from 'styled-components';
import { Logo } from '@components/core/Logo';
import { LOGIN_URL } from '../../../config';

const Container = styled.div`
  width: 100vw;
  height: 100vh;
  align-items: center;
  justify-content: center;
  display: flex;
`;
const LoginBox = styled.div`
  text-align: center;
  height: 538px;
  width: 533px;
  border-radius: 4px;
  background-color: ${({ theme }) => theme.hctPalette.lightBlue};
  box-shadow:
    0 0 2px 0 rgba(0, 0, 0, 0.12),
    0 2px 2px 0 rgba(0, 0, 0, 0.24);
  padding: 50px;
`;
const SubTitle = styled(Typography)`
  && {
    color: #ffff;
    font-size: 24px;
    font-weight: 300;
    line-height: 32px;
    margin-top: ${({ theme }) => theme.spacing(13)};
  }
`;
const LoginButtonContainer = styled.div`
  margin-left: ${({ theme }) => theme.spacing(11)};
  margin-right: ${({ theme }) => theme.spacing(11)};
`;
const LoginButton = styled(Button)`
  && {
    margin-top: ${({ theme }) => theme.spacing(6)};
    width: 100%;
    height: 64px;
    background-color: ${({ theme }) => theme.palette.primary.main};
    color: #ffff;
  }
`;

export function LoginPage(): React.ReactElement {
  const { t } = useTranslation();
  const location = useLocation();
  const params = new URLSearchParams(location.search);
  const next = params.get('next');

  return (
    <Container>
      <LoginBox>
        <Logo transparent={false} displayLogoWithoutSubtitle={false} />
        <SubTitle>{t('Login via Active Directory')}</SubTitle>
        <LoginButtonContainer>
          <LoginButton
            variant="contained"
            size="large"
            //@ts-ignore
            component="a"
            href={
              next
                ? `${LOGIN_URL}?next=/accounts/profile%3Fnext%3D${next}`
                : LOGIN_URL
            }
          >
            {t('Sign in')}
          </LoginButton>
        </LoginButtonContainer>
      </LoginBox>
    </Container>
  );
}
