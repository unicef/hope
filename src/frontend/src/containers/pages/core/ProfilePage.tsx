import * as React from 'react';
import { useEffect } from 'react';
import { useLocation, useNavigate } from 'react-router-dom';
import { useLoggedCheckerQuery } from '@generated/graphql';
import { LOGIN_URL } from '../../../config';

export const ProfilePage = (): React.ReactElement | null => {
  const location = useLocation();
  const navigate = useNavigate();
  const { error } = useLoggedCheckerQuery({ fetchPolicy: 'network-only' });
  const params = new URLSearchParams(location.search);
  const next = params.get('next') ? params.get('next') : '/';

  useEffect(() => {
    if (error) {
      window.location.replace(
        `${LOGIN_URL}?next=${location.pathname}${location.search}`,
      );
    }
    if (next?.startsWith('/api/')) {
      window.location.replace(next);
    }
  }, [error, location, next]);

  useEffect(() => {
    if (!error && next && !next.startsWith('/api/')) {
      navigate(next);
    }
  }, [error, navigate, next]);

  return null;
};
