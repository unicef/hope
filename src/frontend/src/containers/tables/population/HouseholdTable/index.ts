import { HouseholdTable } from './HouseholdTable';

export { HouseholdTable };
