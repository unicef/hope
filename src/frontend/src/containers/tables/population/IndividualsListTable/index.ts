import { IndividualsListTable } from './IndividualsListTable';

export { IndividualsListTable };
