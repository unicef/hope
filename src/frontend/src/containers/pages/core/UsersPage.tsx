import { Button } from '@mui/material';
import * as React from 'react';
import { useState } from 'react';
import { useTranslation } from 'react-i18next';
import { useLocation } from 'react-router-dom';
import { PageHeader } from '@components/core/PageHeader';
import { PermissionDenied } from '@components/core/PermissionDenied';
import { UsersListFilters } from '@components/core/UsersListFilters';
import { hasPermissions, PERMISSIONS } from '../../../config/permissions';
import { usePermissions } from '@hooks/usePermissions';
import { getFilterFromQueryParams } from '@utils/utils';
import { UsersTable } from '../../tables/UsersTable';
import { useBaseUrl } from '@hooks/useBaseUrl';

const initialFilter = {
  search: '',
  partner: '',
  roles: '',
  status: '',
};

export function UsersPage(): React.ReactElement {
  const { businessArea } = useBaseUrl();
  const permissions = usePermissions();
  const { t } = useTranslation();
  const location = useLocation();

  const [filter, setFilter] = useState(
    getFilterFromQueryParams(location, initialFilter),
  );
  const [appliedFilter, setAppliedFilter] = useState(
    getFilterFromQueryParams(location, initialFilter),
  );
  if (permissions === null) return null;

  if (!hasPermissions(PERMISSIONS.USER_MANAGEMENT_VIEW_LIST, permissions))
    return <PermissionDenied />;

  return (
    <>
      <PageHeader title={t('Programme Users')}>
        <>
          <Button
            variant="contained"
            color="primary"
            onClick={() => null}
            component="a"
            href={`/api/download-exported-users/${businessArea}`}
            data-cy="button-target-population-create-new"
          >
            Export
          </Button>
        </>
      </PageHeader>
      <UsersListFilters
        filter={filter}
        setFilter={setFilter}
        initialFilter={initialFilter}
        appliedFilter={appliedFilter}
        setAppliedFilter={setAppliedFilter}
      />
      <UsersTable filter={appliedFilter} />
    </>
  );
}
