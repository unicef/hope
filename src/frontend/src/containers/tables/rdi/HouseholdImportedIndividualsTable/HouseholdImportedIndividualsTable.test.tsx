import * as React from 'react';
import { MockedProvider } from '@apollo/react-testing';
import { act } from 'react';
import wait from 'waait';
import { HouseholdImportedIndividualsTable } from '.';
import { fakeHouseholdChoices } from '../../../../../fixtures/population/fakeHouseholdChoices';
import { fakeImportedHouseholdNode } from '../../../../../fixtures/registration/fakeImportedHouseholdNode';
import { fakeApolloMe } from '../../../../../fixtures/core/fakeApolloMe';
import { ApolloLoadingLink, render } from '../../../../testUtils/testUtils';

describe('containers/tables/rdi/HouseholdImportedIndividualsTable', () => {
  it('should render with data', async () => {
    const { container } = render(
      <MockedProvider mocks={fakeApolloMe}>
        <HouseholdImportedIndividualsTable
          household={fakeImportedHouseholdNode}
          choicesData={fakeHouseholdChoices}
        />
      </MockedProvider>,
    );
    await act(() => wait(0)); // wait for response

    expect(container).toMatchSnapshot();
  });

  it('should render loading', () => {
    const { container } = render(
      <MockedProvider mocks={fakeApolloMe}>
        <HouseholdImportedIndividualsTable
          household={fakeImportedHouseholdNode}
          choicesData={fakeHouseholdChoices}
        />
      </MockedProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
