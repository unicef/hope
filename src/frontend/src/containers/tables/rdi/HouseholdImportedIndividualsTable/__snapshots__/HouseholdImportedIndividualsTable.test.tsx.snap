// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/tables/rdi/HouseholdImportedIndividualsTable should render loading 1`] = `
<div>
  <div
    class="sc-fAUdSK kiWCKq"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-dntaoT dzXcRq css-1ps6pg7-MuiPaper-root"
    >
      <div
        class="MuiTableContainer-root sc-ixGGxD css-rorn0c-MuiTableContainer-root"
      >
        <div
          class="sc-khLCKb snsJA MuiBox-root css-0"
        >
          <div
            class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1w02pvu-MuiToolbar-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6 css-cdr39q-MuiTypography-root"
              data-cy="table-title"
            >
              Individuals in Household
            </h6>
          </div>
          <div
            class="sc-khLCKb snsJA MuiBox-root css-1ruxp1v"
          />
        </div>
        <table
          class="MuiTable-root sc-ivxoEo sc-dstKZu llKqxz ePplQm css-b1kpk9-MuiTable-root"
        >
          <thead
            class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
          >
            <tr
              class="MuiTableRow-root MuiTableRow-head css-n2dwpd-MuiTableRow-root"
            >
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Individual ID
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Individual
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Role
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Relationship to HoH
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeMedium css-1azl6jz-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Date of Birth
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Gender
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Dedupe within Batch
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Dedupe against Population
                </span>
              </th>
            </tr>
          </thead>
          <tbody
            class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
          >
            <tr
              class="MuiTableRow-root MuiTableRow-hover sc-ovuCP gCVigK css-n2dwpd-MuiTableRow-root"
              role="checkbox"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <a
                  class="sc-jtQUzJ atfmU"
                  href="/afghanistan/programs/UHJvZ3JhbU5vZGU6YzRkNTY1N2QtMWEyOS00NmUxLTgxOTAtZGY3Zjg1YTBkMmVm/registration-data-import/individual/SW1wb3J0ZWRJbmRpdmlkdWFsTm9kZTo0YmY3NmE4ZC05MmFjLTQwZjAtOTE2YS04ZDcwMDIwNzFiYmI="
                />
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Al*** Ko***
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Alternate collector
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Not a Family Member. Can only act as a recipient.
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <time
                  datetime="-894672000000"
                >
                  26 Aug 1941
                </time>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Female
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique in batch
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique
              </td>
            </tr>
            <tr
              class="MuiTableRow-root MuiTableRow-hover sc-ovuCP gCVigK css-n2dwpd-MuiTableRow-root"
              role="checkbox"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <a
                  class="sc-jtQUzJ atfmU"
                  href="/afghanistan/programs/UHJvZ3JhbU5vZGU6YzRkNTY1N2QtMWEyOS00NmUxLTgxOTAtZGY3Zjg1YTBkMmVm/registration-data-import/individual/SW1wb3J0ZWRJbmRpdmlkdWFsTm9kZTpjMjZiNmQ2MS1mMTdjLTQ0NmYtYjQ1MC0wMWI0NmMyYmE4Yjk="
                />
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                An*** Ko***
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                None
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Brother / Sister
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <time
                  datetime="947462400000"
                >
                  10 Jan 2000
                </time>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Female
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique in batch
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique
              </td>
            </tr>
            <tr
              class="MuiTableRow-root MuiTableRow-hover sc-ovuCP gCVigK css-n2dwpd-MuiTableRow-root"
              role="checkbox"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <a
                  class="sc-jtQUzJ atfmU"
                  href="/afghanistan/programs/UHJvZ3JhbU5vZGU6YzRkNTY1N2QtMWEyOS00NmUxLTgxOTAtZGY3Zjg1YTBkMmVm/registration-data-import/individual/SW1wb3J0ZWRJbmRpdmlkdWFsTm9kZTozZTE0OGVkYy0wMTY3LTQ0ZTYtOTliYi02ZDA3MmM4ZjA2NWI="
                />
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Ag*** Ko***
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Primary collector
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Head of household (self)
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <time
                  datetime="-188611200000"
                >
                  10 Jan 1964
                </time>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Female
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique in batch
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      <div
        class="MuiTablePagination-root css-jtlhu6-MuiTablePagination-root"
        data-cy="table-pagination"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular MuiTablePagination-toolbar css-wogcmv-MuiToolbar-root-MuiTablePagination-toolbar"
        >
          <div
            class="MuiTablePagination-spacer css-1psng7p-MuiTablePagination-spacer"
          />
          <p
            class="MuiTablePagination-displayedRows css-levciy-MuiTablePagination-displayedRows"
          >
            1–3 of 3
          </p>
          <div
            class="MuiBox-root css-nkn1dz"
          >
            <button
              aria-label="previous page"
              class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
              disabled=""
              tabindex="-1"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowLeftIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.41 16.59 10.83 12l4.58-4.59L14 6l-6 6 6 6z"
                />
              </svg>
            </button>
            <button
              aria-label="next page"
              class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
              disabled=""
              tabindex="-1"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6z"
                />
              </svg>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`containers/tables/rdi/HouseholdImportedIndividualsTable should render with data 1`] = `
<div>
  <div
    class="sc-fAUdSK kiWCKq"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-dntaoT dzXcRq css-1ps6pg7-MuiPaper-root"
    >
      <div
        class="MuiTableContainer-root sc-ixGGxD css-rorn0c-MuiTableContainer-root"
      >
        <div
          class="sc-khLCKb snsJA MuiBox-root css-0"
        >
          <div
            class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1w02pvu-MuiToolbar-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6 css-cdr39q-MuiTypography-root"
              data-cy="table-title"
            >
              Individuals in Household
            </h6>
          </div>
          <div
            class="sc-khLCKb snsJA MuiBox-root css-1ruxp1v"
          />
        </div>
        <table
          class="MuiTable-root sc-ivxoEo sc-dstKZu llKqxz ePplQm css-b1kpk9-MuiTable-root"
        >
          <thead
            class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
          >
            <tr
              class="MuiTableRow-root MuiTableRow-head css-n2dwpd-MuiTableRow-root"
            >
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Individual ID
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Individual
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Role
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Relationship to HoH
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeMedium css-1azl6jz-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Date of Birth
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Gender
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Dedupe within Batch
                </span>
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                scope="col"
              >
                <span
                  class="sc-egkSDF buCIue"
                  data-cy="table-label"
                >
                  Dedupe against Population
                </span>
              </th>
            </tr>
          </thead>
          <tbody
            class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
          >
            <tr
              class="MuiTableRow-root MuiTableRow-hover sc-ovuCP gCVigK css-n2dwpd-MuiTableRow-root"
              role="checkbox"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <a
                  class="sc-jtQUzJ atfmU"
                  href="/afghanistan/programs/UHJvZ3JhbU5vZGU6YzRkNTY1N2QtMWEyOS00NmUxLTgxOTAtZGY3Zjg1YTBkMmVm/registration-data-import/individual/SW1wb3J0ZWRJbmRpdmlkdWFsTm9kZTo0YmY3NmE4ZC05MmFjLTQwZjAtOTE2YS04ZDcwMDIwNzFiYmI="
                />
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Al*** Ko***
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Alternate collector
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Not a Family Member. Can only act as a recipient.
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <time
                  datetime="-894672000000"
                >
                  26 Aug 1941
                </time>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Female
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique in batch
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique
              </td>
            </tr>
            <tr
              class="MuiTableRow-root MuiTableRow-hover sc-ovuCP gCVigK css-n2dwpd-MuiTableRow-root"
              role="checkbox"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <a
                  class="sc-jtQUzJ atfmU"
                  href="/afghanistan/programs/UHJvZ3JhbU5vZGU6YzRkNTY1N2QtMWEyOS00NmUxLTgxOTAtZGY3Zjg1YTBkMmVm/registration-data-import/individual/SW1wb3J0ZWRJbmRpdmlkdWFsTm9kZTpjMjZiNmQ2MS1mMTdjLTQ0NmYtYjQ1MC0wMWI0NmMyYmE4Yjk="
                />
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                An*** Ko***
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                None
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Brother / Sister
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <time
                  datetime="947462400000"
                >
                  10 Jan 2000
                </time>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Female
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique in batch
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique
              </td>
            </tr>
            <tr
              class="MuiTableRow-root MuiTableRow-hover sc-ovuCP gCVigK css-n2dwpd-MuiTableRow-root"
              role="checkbox"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <a
                  class="sc-jtQUzJ atfmU"
                  href="/afghanistan/programs/UHJvZ3JhbU5vZGU6YzRkNTY1N2QtMWEyOS00NmUxLTgxOTAtZGY3Zjg1YTBkMmVm/registration-data-import/individual/SW1wb3J0ZWRJbmRpdmlkdWFsTm9kZTozZTE0OGVkYy0wMTY3LTQ0ZTYtOTliYi02ZDA3MmM4ZjA2NWI="
                />
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Ag*** Ko***
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Primary collector
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Head of household (self)
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                <time
                  datetime="-188611200000"
                >
                  10 Jan 1964
                </time>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Female
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique in batch
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
              >
                Unique
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      <div
        class="MuiTablePagination-root css-jtlhu6-MuiTablePagination-root"
        data-cy="table-pagination"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular MuiTablePagination-toolbar css-wogcmv-MuiToolbar-root-MuiTablePagination-toolbar"
        >
          <div
            class="MuiTablePagination-spacer css-1psng7p-MuiTablePagination-spacer"
          />
          <p
            class="MuiTablePagination-displayedRows css-levciy-MuiTablePagination-displayedRows"
          >
            1–3 of 3
          </p>
          <div
            class="MuiBox-root css-nkn1dz"
          >
            <button
              aria-label="previous page"
              class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
              disabled=""
              tabindex="-1"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowLeftIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.41 16.59 10.83 12l4.58-4.59L14 6l-6 6 6 6z"
                />
              </svg>
            </button>
            <button
              aria-label="next page"
              class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
              disabled=""
              tabindex="-1"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6z"
                />
              </svg>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
